core/src/main/java/com/amplifyframework/core/model/query/QuerySortBy.java

```diff
@@ -0,0 +1,92 @@
+/*
+ * Copyright 2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License").
+ * You may not use this file except in compliance with the License.
+ * A copy of the License is located at
+ *
+ *  http://aws.amazon.com/apache2.0
+ *
+ * or in the "license" file accompanying this file. This file is distributed
+ * on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ * express or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+
+package com.amplifyframework.core.model.query;
+
+import androidx.annotation.NonNull;
+import androidx.core.util.ObjectsCompat;
+
+import com.amplifyframework.core.Consumer;
+import com.amplifyframework.core.model.query.predicate.QueryField;
+import com.amplifyframework.datastore.DataStoreCategoryBehavior;
+
+import java.util.Objects;
+
+/**
+ * A data structure representing a model field and an order to sort by (ascending or descending), used to specify the
+ * order of results from {@link DataStoreCategoryBehavior#query(Class, QueryOptions, Consumer, Consumer)}.
+ *
+ * The preferred way to create a QuerySortBy is with the @{@link QueryField#ascending()} and
```

Comment:
Should the first "@" in "@{@link QueryField#ascending()}" be removed?

Reply:
Good catch!